src/components/Avatar.js

```diff
@@ -23,6 +24,9 @@ const propTypes = {
 
     /** The fill color for the icon. Can be hex, rgb, rgba, or valid react-native named color such as 'red' or 'blue' */
     fill: PropTypes.string,
+
+    /** Function for using fallback avatar */
```

Comment:
This doesn’t make any sense, can we add something more meaningful!

Reply:
@Santhosh-Sellavel does this looks good ?

`A fallback avatar component to display when there is an error on loading remote URL`

Reply:
`A fallback avatar icon to display when there is an error on loading avatar from remote URL.`

Reply:
@AndrewGable Any thoughts?